src/shared/utils/index.ts

```diff
@@ -204,3 +204,51 @@ export function blacklistKeys(raw: { [key: string]: any }, keys: string[]) {
       return obj;
     }, {});
 }
+
+/**
+ * True if given date is today, false otherwise
+ */
+export function isToday(date: string): boolean {
+  const today = new Date(Date.now());
+  const givenDate = new Date(date);
+
+  if (givenDate.toString() === 'Invalid Date') {
+    return false;
+  }
+  return (
+    givenDate.getDay() === today.getDay() &&
+    givenDate.getMonth() === today.getMonth() &&
+    givenDate.getFullYear() === today.getFullYear()
+  );
+}
+
+/**
+ * How long ago from today in days
+ *
+ * return -1 if given date if from the future
+ *
+ */
+export function howLongAgo(date: string): number {
+  const today = new Date(Date.now());
+  const givenDate = new Date(date);
+  const millisecondsInDay = 24 * 60 * 60 * 1000;
+
+  // reset hours for an accurate comparison
+  today.setHours(0, 0, 0, 0);
+  givenDate.setHours(0, 0, 0, 0);
+  // + operator to make date a number
+  return (+today - +givenDate) / millisecondsInDay;
+}
+
+export function displayTimeFromDate(date: string): string {
```

Comment:
Do you want to display time in UTC, or in the local timezone of the user (when it's server-side rendered, it will be the timezone of the server), or in the timezone of whatever was stored in Nexus for this resource?